vault/logical_system.go

```diff
@@ -1521,6 +1522,15 @@ func (b *SystemBackend) handleMount(ctx context.Context, req *logical.Request, d
 		config.IdentityTokenKey = apiConfig.IdentityTokenKey
 	}
 
+	// Don't lazily generate the default OIDC key for KV mounts. A default KV mount
```

Comment:
TIL, I wouldn't have thought the HTTP mount handler function would be in the call path for the default KV mount.